### ISSUE TITLE
Role attribute (role="img") fails WCAG audit

### ISSUE THREAD

**Issue by kraenels** [FortAwesome/angular-fontawesome#148]
I'm running a WCAG audit on a web app, which uses the menu bars icon as well as several others in sidebar menu items (e.g., home icon for the Home item). The app is currently failing the audit because the `<fa-icon>` component builds the svg with the 'role="img"' property.

Here's the requirement from WCAG, provided by Siteimprove's Google Chrome extension:

1.1.1 Non-text Content
The WAI-ARIA image, created with role="img", does not have an accessible name.

If the image is informative or functional, use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to create an alternative text for the image.

If the image is purely decorative, use the WAI-ARIA attribute role="presentation" instead of role="img" for the image. This will cause assistive technologies to ignore the image.

I would like to be able to assign an aria-label directly to the inner SVG from the `<fa-icon>` component.

**Comment by mlwilkerson** [comment]
Thanks for the report.

If there is an accessibility correction to be made along these lines, then it might actually need to be made deeper in our stack, such as in the `fontawesome-svg-core` package (and maybe elsewhere too).

@robmadole thoughts?

**Comment by robmadole** [comment]
Yes the best solution here is to allow the `role` to be passed in as a property when rendering. This way it can be changed from the default of `role="img"`. This is already supported in the `@fortawesome/fontawesome-svg-core` library which probably means we just need to update `angular-fontawesome` to support it.

**Comment by devoto13** [comment]
`title` input of the `fa-icon` allows to set an alternative text for the icon (it will add `aria-labelledby` attribute). I'll add possibility to customise the `role` attribute of the SVG as well.

**Comment by kraenels** [comment]
Thanks, @devoto13!
